packages/apps-config/src/endpoints/production.ts

```diff
@@ -198,8 +198,8 @@ export const prodChains: EndpointOption[] = [
     text: 'Nodle',
     providers: {
       // Nodle: 'wss://main3.nodleprotocol.io', // https://github.com/polkadot-js/apps/issues/7652
-      OnFinality: 'wss://nodle.api.onfinality.io/public-ws'
-      // Pinknode: 'wss://rpc.pinknode.io/nodle/explorer' // https://github.com/polkadot-js/apps/issues/5721
+      OnFinality: 'wss://nodle.api.onfinality.io/public-ws',
+      Pinknode: 'wss://public-rpc.pinknode.io/nodle'
```

Comment:
Duplicated under Polkadot parachains. 

The production chain is not the same as the parachain - these should be 2 different networks - https://github.com/polkadot-js/apps/runs/6806069215?check_suite_focus=true#step:3:710

Reply:
Removed the endpoint from here.